### PR TITLE
Fortanix is a friend of Rust

### PR DESCRIPTION
**Pull request by jethrogb**

### CODE REVIEW

**Comment by brson** [comment]
Thanks @jethrogb! I'm glad to see what you've been up to.

**Comment by brson** [comment]
The travis failure _seems_ legit, but I can't quite tell what's wrong from looking at the source.

File: _data/users.yml [hunk 1]
```diff
@@ -178,6 +178,11 @@
   logo: faraday.svg
   how: "<a href='https://github.com/faradayio/credentials_to_env'>Securely transferring credentials.</a>"
 -
+  name: Fortanix:
```
Comment (alexcrichton) [inline _data/users.yml]: I think the trailing colon here may be causing problems? If the key is surrounded in quotes though I think it's valid yml still?

Comment (jethrogb) [inline _data/users.yml]: Whoops, fixed

**Comment by alexcrichton** [comment]
👍

**Pull request merged**